Gemfile.lock

```diff
@@ -134,9 +142,7 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    calculated_attributes (0.5.0)
-      activerecord (>= 3.2.20)
-    cancancan (3.6.1)
+    cancancan (3.5.0)
```

Comment:
`cancancan` downgraded?

Reply:
sorry, it's because of the `bundle update` command that I used to automatically renew the version of our gems, but apparently some got downgraded even though I manually upgrade it later on

will squash all the commits for 6.1 into 1